openioc-to-stix.py

```diff
@@ -11,22 +11,15 @@
 import argparse
 
 # python-stix
-from stix import utils
-from stix.indicator import Indicator
-from stix.core import STIXPackage, STIXHeader
-from stix.common import InformationSource
-from stix.common.vocabs import PackageIntent
-
-# python-cybox
-from cybox.core import Observables
-from cybox.common import ToolInformationList, ToolInformation
-
-# internal
+from openioc2stix import translate
```

Comment:
Let's order the imports in the way recommended by [pep8](https://www.python.org/dev/peps/pep-0008/#imports), so  we'd end up with something like:

```
import argparse 

import stix.utils

from openioc2stix import translate
from openioc2stix.version import __Version__